WebRoot/js/zimbraAdmin/accounts/view/ZaAccountXFormView.js

```diff
@@ -3287,7 +3288,8 @@ textFieldCssClass:"admin_xform_number_input"}
                         			ZaAccount.A_zimbraPasswordMinDigitsOrPuncs,
                         			ZaAccount.A_zimbraMinPwdAge,
                         			ZaAccount.A_zimbraMaxPwdAge,
-                        			ZaAccount.A_zimbraEnforcePwdHistory]]],
+                                    ZaAccount.A_zimbraEnforcePwdHistory,
```

Comment:
check indentation in entire PR

Reply:
resolved but maintained parity with existing code.